concent_api/core/tests/test_payments_bankster.py

```diff
@@ -269,3 +258,108 @@ def test_that_in_not_additional_verification_use_case_provider_claim_payment_inf
+        )
+
+        with freeze_time("2018-02-05 10:00:15"):
+            subtask_results_accepted_list = [
+                self._get_deserialized_subtask_results_accepted(
+                    task_to_compute=task_to_compute
+                )
+            ]
+
+        with freeze_time("2018-02-05 10:00:25"):
+            with mock.patch(
+                'core.payments.bankster.service.get_deposit_value',
+                return_value=0
+            ) as get_deposit_value_mock:
+                requestors_claim_payment_info = settle_overdue_acceptances(
+                    requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
+                    provider_ethereum_address=task_to_compute.provider_ethereum_address,
+                    acceptances=subtask_results_accepted_list,
+                )
+
+        get_deposit_value_mock.assert_called()
+
+        self.assertIsNone(requestors_claim_payment_info.tx_hash)
+        self.assertIsNone(requestors_claim_payment_info.payment_ts)
+        self.assertEqual(requestors_claim_payment_info.amount_paid, 0)
+        self.assertEqual(requestors_claim_payment_info.amount_pending, task_to_compute.price)
+
+    @override_settings(
+        PAYMENT_DUE_TIME=10
+    )
+    def test_that_settle_overdue_acceptances_should_return_claim_deposit_info_with_amount_paid(self):
+        task_to_compute = self._get_deserialized_task_to_compute(
+            price=15000,
+        )
+
+        with freeze_time("2018-02-05 10:00:15"):
+            subtask_results_accepted_list = [
+                self._get_deserialized_subtask_results_accepted(
+                    task_to_compute=task_to_compute
+                )
+            ]
+
+        with freeze_time("2018-02-05 10:00:25"):
+            with mock.patch('core.payments.bankster.service.get_deposit_value', return_value=1000) as get_deposit_value_mock:
+                with mock.patch(
+                    'core.payments.bankster.service.get_list_of_payments',
+                    side_effect=[
+                        self._get_list_of_batch_transactions(),
+                        self._get_list_of_force_transactions(),
+                    ]
+                ) as get_list_of_payments_mock:
+                    requestors_claim_payment_info = settle_overdue_acceptances(
+                        requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
+                        provider_ethereum_address=task_to_compute.provider_ethereum_address,
+                        acceptances=subtask_results_accepted_list,
+                    )
+
+        get_deposit_value_mock.assert_called_once()
+        get_list_of_payments_mock.assert_called()
+
+        self.assertIsNotNone(requestors_claim_payment_info.tx_hash)
+        self.assertIsNotNone(requestors_claim_payment_info.payment_ts)
+        self.assertEqual(requestors_claim_payment_info.amount_paid, 1000)
+        self.assertEqual(requestors_claim_payment_info.amount_pending, 1000)
```

Comment:
shouldn't `amount_pending` be equal to 1500 (price) - 1000 (amount_paid) = 14000?

Reply:
No, please check `_get_list_of_batch_transactions` and `_get_list_of_force_transactions` mock functions in this tests. Calculations are like this:

```
already_paid_value = 15000 - (
    (1000 + 2000 + 3000 + 4000)(batch_transactions) +
    (1000 + 2000)(force_transactions) +
)
already_paid_value == 13000, so 2000 left
get_deposit_value returns 1000, so 1000 paid and 1000 left (pending)
```

Reply:
can you please put in in a comment (in a short from)? It is not visible at a first glance

Reply:
:+1: done